pixeltable/env.py

```diff
@@ -571,10 +571,28 @@ def _init_metadata(self) -> None:
         metadata.create_system_info(self._sa_engine)
 
     def _create_engine(self, time_zone_name: str, echo: bool = False) -> None:
-        connect_args = {'options': f'-c timezone={time_zone_name}'}
-        self._logger.info(f'Creating SQLAlchemy engine with connection arguments: {connect_args}')
+        # Parse the database URL
+        db_url = sql.make_url(self.db_url)
+
+        # Get existing options and parse them
+        # Query parameters can be strings or tuples (if multiple values exist)
+        existing_options_raw = db_url.query.get('options', '') if db_url.query else ''
+        existing_options = (
+            ' '.join(existing_options_raw) if isinstance(existing_options_raw, tuple) else existing_options_raw
+        )
+        option_parts = existing_options.split() if existing_options else []
```

Comment:
Doesn't this just reverse the preceding operation (join the tuple only to split it again)?

Why not simply
```python
option_parts = list(existing_options_raw) if isinstance(existing_options_raw, tuple) else existing_options_raw.split()
```
If that doesn't work please add a code comment explaining why we need the 2-step parsing